src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Items/ReplaceContentDefinitionDeploymentStep.Fields.Summary.cshtml

```diff
@@ -32,12 +32,13 @@ else
 
     if (contentParts?.Length > 0)
     {
-        foreach (var part in avaParts)
+        foreach (var part in parts)
         {
             if (contentParts.Contains(part.Name))
             {
-                <span class="badge badge-dark">@part.Name</span>
+                <span class="badge ta-badge">@part.Name</span>
             }
         }
     }
 }
+<span class="hint">@T["Content Definitions should be placed before any content steps."]</span>
```

Comment:
Can you please go to a new line after the tags?

![image](https://user-images.githubusercontent.com/703248/108193479-c51a0200-7115-11eb-8bb6-cc9c393bf97e.png)